Resources/Audio/Misc/attributions.yml

```diff
@@ -1,4 +1,15 @@
-- files: ["tension_session.ogg"]
+- files: ["emergency_meeting.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Taken from TG station."
+  source: "https://github.com/tgstation/tgstation/commit/97945e7d08d1457ffc27e46526a48c0453cc95e4"
```

Comment:
Isn't this an unlicensed sound taken from among us?

Reply:
not sure why this was added tbh, i dont need it and will remove.